be/src/storage/delta_writer.cpp

```diff
@@ -248,6 +248,10 @@ Status DeltaWriter::close() {
 }
 
 Status DeltaWriter::_flush_memtable_async() {
+    // _mem_table is nullptr means write() has not been called
+    if (_mem_table == nullptr) {
+        return Status::OK();
```

Comment:
If you return here, will the _flush_token work fine?

Reply:
it work well, _flush_token->wait() will return if there are has no task submit